library/src/commonTest/kotlin/com/ioki/passenger/api/models/ApiRideInquiryRequestTest.kt

```diff
@@ -20,26 +20,111 @@ internal class ApiRideInquiryRequestTest : IokiApiModelTest() {
+                            ),
+                            ApiOption(
+                                slug = "public_transport_ticket",
+                                value = AnyValue.BooleanValue(true),
+                            ),
+                            ApiOption(
+                                slug = "blue_badge",
+                                value = AnyValue.BooleanValue(false),
+                            ),
+                        ),
+                        firstName = null,
+                        lastName = null,
+                    ),
+                    ApiPassengerSelectionRequest(
+                        type = "child",
+                        options = listOf(
+                            ApiOption(
+                                slug = "wheelchair",
+                                value = AnyValue.BooleanValue(true),
+                            ),
+                            ApiOption(
+                                slug = "bahncard",
+                                value = AnyValue.BooleanValue(true),
+                            ),
+                            ApiOption(
+                                slug = "walker",
+                                value = AnyValue.BooleanValue(false),
+                            ),
+                            ApiOption(
+                                slug = "public_transport_ticket",
+                                value = AnyValue.BooleanValue(false),
+                            ),
+                            ApiOption(
+                                slug = "blue_badge",
+                                value = AnyValue.BooleanValue(true),
+                            ),
+                        ),
+                        firstName = null,
+                        lastName = null,
+                    ),
                 ),
             ),
-            rideInquiryRequest,
+            rideInquiryJson,
+        )
+    }
+
+    @Test
+    fun serializationMinimal() {
+        testSerializationWithJsonString(
+            ApiRideInquiryRequest(
+                productId = "prd_123",
+                origin = ApiRideInquiryRequest.Location(
+                    lat = 1.0,
+                    lng = 2.0,
+                    locationName = "location name",
+                    streetName = "street name",
+                    streetNumber = "17B",
+                    postalCode = "12345",
+                    city = "city",
+                    county = "county",
+                    country = "country",
+                    time = Instant.parse("1970-01-01T00:00:00Z"),
+                    stationId = "station id",
+                ),
+                destination = defaultApiLocation(),
+                passengers = null,
+            ),
+            rideInquiryMinimalJson,
         )
     }
 }
 
-private val rideInquiryRequest =
+private fun defaultApiLocation(): ApiRideInquiryRequest.Location = ApiRideInquiryRequest.Location(
```

Comment:
1. Don't we have the `Helper` files for these stubs? 🤔 
2. I guess in API tests we shouldn't do that. But 🤷‍♂️ 🙃

Reply:
ping @vhontar